src/main/scala/li/cil/oc/integration/vanilla/ConverterFluidContainerItem.scala

```diff
@@ -0,0 +1,37 @@
+package li.cil.oc.integration.vanilla
+
+import java.util
+
+import li.cil.oc.{Settings, api}
+import net.minecraft.item
+import net.minecraft.item.ItemStack
+import net.minecraftforge.fluids
+import net.minecraftforge.fluids.FluidStack
+
+import scala.collection.convert.WrapAsScala._
+
+object ConverterFluidContainerItem extends api.driver.Converter  {
+  override def convert(value: AnyRef, output: util.Map[AnyRef, AnyRef]): Unit =
```

Comment:
I was wandering, if there any specific reason why converters do return parameters instead of normal return?

Reply:
Efficiency? Also you may want to examine this map for the other convertres results if you know they are registerd before.